__docs__/wonder-blocks-dropdown/multi-select.argtypes.ts

```diff
@@ -17,15 +29,15 @@ const argTypes: ArgTypes = {
     labels: {
         control: {type: "object"},
         description:
-            "The object containing the custom labels used inside this component.",
+            "The object containing the custom values used inside this component.",
         table: {
             type: {summary: "Labels"},
         },
     },
     showOpenerLabelAsText: {
         control: {type: "boolean"},
-        description: `When false, the SelectOpener can show a Node as a label. When true, the
-         SelectOpener will use a string as a label. If using custom OptionItems, a
+        description: `When false, the SelectOpener can show a Node as a value. When true, the
+         SelectOpener will use a string as a value. If using custom OptionItems, a
```

Comment:
Same thing from the singleselect docs!
```suggestion
        description: `When false, the SelectOpener can show a Node as a selected value. When true, the
         SelectOpener will use a string as a selected value. If using custom OptionItems, a
```